docs/releasing/publishing.md

```diff
@@ -43,17 +44,17 @@ Developers should pair on releases. When remote working, it can be useful to be
 
 3. Run `npm run publish-release`, which will prompt you to check whether the npm tag looks as expected.
 
-  If you're following these instructions, you're probably doing a sequential release, meaning
-  the tag should be 'latest'.
+If you're following these instructions, you're probably doing a sequential release, meaning
+the tag should be 'latest'.
 
 4. Enter `y` to continue. If you think the tag should be different, enter `N` to have the option to set your own npm tag.
 
 5. You will now be prompted to continue or cancel the release. Check the details and enter `y` to continue. If something does not look right, press `N` to cancel the release.
-  
-  This step will create a ZIP file containing the release in the root of your govuk-frontend git directory. You will need this file when creating the GitHub release.
 
-  It will also automatically create a tag in Github which you can use to create a Github release in the following section.
-  
+This step will create a ZIP file containing the release in the root of your govuk-frontend git directory. You will need this file when creating the GitHub release.
+
+It will also automatically create a tag in Github which you can use to create a Github release in the following section.
```

Comment:
```suggestion
   This step will create a ZIP file containing the release in the root of your govuk-frontend git directory. You will need this file when creating the GitHub release.

   It will also automatically create a tag in Github which you can use to create a Github release in the following section.
```

Reply:
Sorted 👍